src/app/App.tsx

```diff
@@ -1,7 +1,45 @@
 import React from 'react';
+import styles from './App.module.css';
+import Title from './components/Title/Title';
 
 function App(): JSX.Element {
-  return <main>Hello Cologne</main>;
+  return (
+    <main className={styles.container}>
+      <div>
+        <Title />
+        <form className={styles.form}>
+          <select>
+            <option>Select participant</option>
+            <option>Alice P</option>
+            <option>Alice S</option>
+            <option>Manuel F</option>
+            <option>Riitta</option>
+          </select>
+          or create new
+          <input
+            className={styles.textinput}
+            type="text"
+            placeholder="first name"
+          />
+          <input
+            className={styles.textinput}
+            type="text"
+            placeholder="last name"
+          />
+          <input
+            className={styles.submitbutton}
+            type="submit"
+            value="register"
```

Comment:
this doesn't seem to work, try: ` value={"register"}`

Reply:
sorry, old picture in pull request. Code works :)

Reply:
Sorry, I was wrong about that